.env

```diff
@@ -1,3 +1,3 @@
-PORT=8034
+PORT=8043
```

Comment:
Why are we changing the port?

Reply:
Yeah, that does look like a mistake. 8043 is actually Anbox. Thanks